CHANGELOG.md

```diff
@@ -0,0 +1,53 @@
+# Changelog
+
+
+## Version 2.0.0-alpha.2
+
+- Font family and size selection in Preferences (Edit menu)
```

Comment:
It's not in the `Edit` menu for Macs. It's in the Application menu. In this case, `Python`, hopefully if we can package it properly, under the `Guiguts` menu once we get a formal release.

Reply:
I'll edit it to mention the Python menu on Macs.